charts/astronomer/templates/houston/worker/houston-worker-configmap.yaml

```diff
@@ -0,0 +1,18 @@
+######################################
```

Comment:
@andriisoldatenko or @sjmiller609 I'm not 100% if we need this or not.

Reply:
it's just comment.

Reply:
###################################
## Astronomer Houston Worker ConfigMap ##
###################################
```

Reply:
I mean, I don't know if we need this entire file.